topydo.conf

```diff
@@ -6,6 +6,8 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+console_height              = 25
```

Comment:
These options are static and not really flexible, I'd suggest to read the console size dynamically at runtime. For example:

```python
rows, columns = subprocess.check_output(['stty', 'size']).split()
```

[Source](https://stackoverflow.com/questions/566746/how-to-get-console-window-width-in-python)